instrumentation/opentelemetry-instrumentation-psycopg2/pyproject.toml

```diff
@@ -34,6 +34,7 @@ dependencies = [
 [project.optional-dependencies]
 instruments = [
   "psycopg2 >= 2.7.3.1",
+  "psycopg2-binary >= 2.7.3.1",
```

Comment:
Is this what we want tho?

Using `pip install opentelemetry-instrumentation-psycopg2[instruments]` would install both `psycopg2-binary` and `psycopg2`.

Reply:
We are abusing optional dependencies to do discovery of things, I hope the `opentelementry-instrumentation-x[instruments]` is not documented anywhere.